SignalMessaging/utils/ThreadUtil.m

```diff
@@ -185,8 +185,8 @@ + (TSOutgoingMessage *)buildOutgoingMessageWithText:(nullable NSString *)fullMes
     BOOL isVoiceMessage = (attachments.count == 1 && attachments.lastObject.isVoiceMessage);
     
     // Loki: If we're not friends then always set the message to a friend request message
-    // If we're friends then the assumption is that we have the other user's pre-key bundle
-    TSThreadFriendRequestStatus friendRequestStatus = [thread getFriendRequestStatusWithTransaction:transaction];
+    // If we're friends then the assumption is that we have the other user's prekey bundle
+    TSThreadFriendRequestStatus friendRequestStatus = thread.friendRequestStatus;
     BOOL isFriend = friendRequestStatus == TSThreadFriendRequestStatusFriends;
```

Comment:
maybe better to have a function in thread? `thread.isFriend`

Reply:
I felt like it wasn't necessary because this is the only place where this is currently used. Happy to add it though